openghg/store/_footprints.py

```diff
@@ -384,7 +385,7 @@ def read_file(
         # These are the keys we will take from the metadata to search the
         # metadata store for a Datasource, they should provide as much detail as possible
         # to uniquely identify a Datasource
-        required = ("site", "model", "inlet", "domain")
+        required = ("site", "model", "inlet", "domain", "time_resolution", "spatial_resolution")
```

Comment:
Is there a reason not to also add "short_lifetime" (or whatever the boolean equivalent we were discussing was)?